### PR TITLE
Replace cheshire with clojure.data.json

### PR DESCRIPTION
**Pull request by dpassen**
- Keep Jackson conflicts far away

Addresses #101

### CODE REVIEW

**Comment by vemv** [comment]
You were quicker than me :) LGTM

File: deps.edn [hunk 1]
```diff
@@ -5,7 +5,7 @@
   org.clojure/data.zip {:mvn/version "1.0.0"}
   org.clojure/tools.cli {:mvn/version "1.0.206"}
   org.clojure/tools.deps.alpha {:mvn/version "0.12.1003"}
-  cheshire/cheshire {:mvn/version "5.10.1"}
+  org.clojure/data.json {:mvn/version "2.4.0"}
```
Comment (dpassen) [inline deps.edn:8]: If this is meant to be kept alphabetized (within org.clojure), easy change

Comment (liquidz) [reply deps.edn:8]: No problem :)

**Review by liquidz — APPROVED** [review]
@dpassen Sorry for late reaction.
LGTM. Thanks!

**Comment by dpassen** [comment]
@liquidz Thank you. Hope the second vaccine didn't hit you too hard :-)

**Pull request merged**